code/game/machinery/bots/mulebot.dm

```diff
@@ -390,23 +391,19 @@ var/global/mulebot_count = 0
 	else
 		to_chat(user, "<span class='warning'>Access denied.</span>")
 		return 0
-
-// mousedrop a crate to load the bot
 // can load anything if emagged
 
-/obj/machinery/bot/mulebot/MouseDrop_T(var/atom/movable/C, mob/user)
-
-	if(user.stat)
-		return
 
-	if (!on || !istype(C)|| C.anchored || get_dist(user, src) > 1 || get_dist(src,C) > 1 )
-		return
+// Mousedrop a crate to load the bot
 
-	if(load)
-		return
+/obj/machinery/bot/mulebot/MouseDrop_T(atom/dropping, mob/user)
+	if(user.stat)
```

Comment:
user.stat means that the user is unconscious. You want to check that the guy is NOT unconscious.
You also should do "if(!user.incapacitated ())", which checks for handcuffs and stuns and shit as well as unconsciousness.